src/platform/haswell/include/platform/platform.h

```diff
@@ -65,6 +65,7 @@ struct sof;
 #define PLATFORM_IRQ_TASK_HIGH	IRQ_NUM_SOFTWARE2
 #define PLATFORM_IRQ_TASK_MED	IRQ_NUM_SOFTWARE1
 #define PLATFORM_IRQ_TASK_LOW	IRQ_NUM_SOFTWARE1
+#define PLATFORM_IRQ_TASK_LEVEL 2
```

Comment:
Wouldn't it be better to remov TASK_MED level on BDW/HSW and just use High/Low priorities ? You could use conditional compilation to remove TASK_MED out of generic code when compiled with CONFIG_HSW

Reply:
yes, good idea